src/range.rs

```diff
@@ -59,10 +65,7 @@ impl TextRange {
     /// Create a range up to the given end (`..end`).
     #[inline]
     pub const fn up_to(end: TextSize) -> TextRange {
-        TextRange {
-            start: TextSize::zero(),
-            end,
-        }
+        TextRange { start: TextSize(0), end }
```

Comment:
Is there a reason to change this? FWIW, `TextRange(TextSize::zero(), end)` does stay on one line.